### PR TITLE
Added cool guides from github repos

### PR DESCRIPTION
**Pull request by fklement**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -41,3 +41,9 @@ Note: This does not include courses primarily based on YouTube
   - [Codecademy (Freemium)](https://www.codecademy.com)
 ### Guides
   - [P1xt's Full Computer Science Guide](https://github.com/P1xt/p1xt-guides)
+  - [30-seconds-of-code](https://github.com/30-seconds/30-seconds-of-code)
+  - [iOSBridge](https://github.com/mobilebridge/iosbridge)
+  - [freeCodeCamp](https://github.com/freeCodeCamp/freeCodeCamp)
```
Comment (b3u) [inline README.md]: see line 40

File: README.md [hunk 2]
```diff
@@ -41,3 +41,9 @@ Note: This does not include courses primarily based on YouTube
...
+  - [go](https://github.com/datasciencemasters/go)
```
Comment (b3u) [inline README.md]: is this a guide or curriculum?

Comment (fklement) [reply README.md]: This are only guides.

File: README.md (hunk 1 above)
Comment (b3u) [reply README.md]: please remove freecodecamp, as it is already included

File: README.md (hunk 2 above)
Comment (b3u) [reply README.md]: :+1:

**Comment by b3u** [comment]
@fklement see #9 please

**Pull request merged**